src/rabbit_misc.erl

```diff
@@ -80,6 +80,7 @@
 -export([report_default_thread_pool_size/0]).
 -export([get_gc_info/1]).
 -export([group_proplists_by/2]).
+-export([get_erl_path/0]).
```

Comment:
I think this belongs to `rabbit_runtime`.

Reply:
Sure thing, changing it now.